seed_db.sh

```diff
@@ -123,15 +123,15 @@ else
     docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (4, 'ENGLISH_UK', 'TRANSLATE', 4);"
 
     docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'GERMAN', 'TRANSLATE', 2);"
-    docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'ENGLISH_US', 'TRANSLATE', 1);"
+    docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'ENGLISH_UK', 'TRANSLATE', 1);"
```

Comment:
the `user_id` for this entry should be changed to 4 since user 1 doesn't have approval for `ENGLISH_UK`